src/instructlab/configuration.py

```diff
@@ -507,6 +507,11 @@ class _train(BaseModel):
         ge=0,
         description="Number of samples the model should see before saving a checkpoint during phase1. Disabled when set to 0.",
     )
+    phased_phase1_learning_rate: float = Field(
+        default=2e-5,
+        ge=0,
```

Comment:
what is ge? I have never seen this before in a flag

Reply:
greater than or equal to

Reply:
and gt is just greater than